Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -188,40 +190,73 @@ void NANDImporter::ProcessFile(const NANDFSTEntry& entry, const std::string& par
   }
 }
 
-void NANDImporter::ExtractCertificates(const std::string& nand_root)
+bool NANDImporter::ExtractCertificates(const std::string& nand_root)
 {
-  const std::string title_contents_path =
-      nand_root + "/title/00000001/0000000d/content/00000011.app";
-  File::IOFile file(title_contents_path, "rb");
-  if (!file)
+  const std::string content_dir = nand_root + "/title/00000001/0000000d/content/";
+
+  File::IOFile tmd_file(content_dir + "title.tmd", "rb");
+  std::vector<u8> tmd_bytes(tmd_file.GetSize());
+  if (!tmd_file.ReadBytes(tmd_bytes.data(), tmd_bytes.size()))
   {
-    PanicAlertT("Unable to open %s! Refer to "
-                "https://dolphin-emu.org/docs/guides/wii-network-guide/ to set up "
-                "certificates.",
-                title_contents_path.c_str());
-    return;
+    ERROR_LOG(DISCIO, "ExtractCertificates: Could not read IOS13 TMD");
+    return false;
+  }
+
+  IOS::ES::TMDReader tmd(std::move(tmd_bytes));
+  IOS::ES::Content content_metadata;
+  if (!tmd.GetContent(tmd.GetBootIndex(), &content_metadata))
+  {
+    ERROR_LOG(DISCIO, "ExtractCertificates: Could not get content ID from TMD");
+    return false;
+  }
+
+  File::IOFile content_file(content_dir + StringFromFormat("%08x.app", content_metadata.id), "rb");
+  std::vector<u8> content_bytes(content_file.GetSize());
+  if (!content_file.ReadBytes(content_bytes.data(), content_bytes.size()))
+  {
+    ERROR_LOG(DISCIO, "ExtractCertificates: Could not read IOS13 contents");
+    return false;
   }
 
-  struct Certificate
+  struct PEMCertificate
   {
-    u32 offset;
-    u32 size;
     std::string filename;
+    std::array<u8, 4> search_bytes;
   };
-  std::array<Certificate, 3> certificates = {{{0x92834, 1005, "/clientca.pem"},
-                                              {0x92d38, 609, "/clientcakey.pem"},
-                                              {0x92440, 897, "/rootca.pem"}}};
-  for (const Certificate& cert : certificates)
+
+  std::array<PEMCertificate, 3> certificates = {{
+      {"/clientca.pem", {{0x30, 0x82, 0x03, 0xE9}}},
+      {"/clientcakey.pem", {{0x30, 0x82, 0x02, 0x5D}}},
+      {"/rootca.pem", {{0x30, 0x82, 0x03, 0x7D}}},
```

Comment:
These bytes are ASN.1 encoding of the cert. You could put some comment but implementing a full-blown ASN.1 parser would of course be silly :P